com.github.qarmin.szyszka.yaml

```
@@ -0,0 +1,36 @@
+app-id: com.github.qarmin.szyszka
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: szyszka
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--filesystem=/home"
+- "--filesystem=/mnt"
+- "--filesystem=/media"
+- "--filesystem=/run/media"
+- "--device=dri"
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin"
+  env:
+    CARGO_HOME: "/run/build/szyszka/cargo"
+modules:
+- name: szyszka
+  buildsystem: simple
+  build-commands:
+  - cargo --offline fetch --manifest-path Cargo.toml
+  - cargo --offline build --release
+  - install -Dm755 ./target/release/szyszka -t /app/bin/
+  - install -Dm644 ./data/icons/com.github.qarmin.szyszka.svg -t /app/share/icons/hicolor/scalable/apps/
+  - install -Dm644 ./data/com.github.qarmin.szyszka.desktop -t /app/share/applications/
+  - install -Dm644 ./data/com.github.qarmin.szyszka.metainfo.xml -t /app/share/metainfo
+  sources:
+  - cargo-sources.json
+  - type: git
+    url: https://github.com/qarmin/szyszka.git
+    tag: 2.0.0
+    sha: 827b94496f09f0d0bea14e0d2f44bdf0496b1f76
```

It's commit" `(flatpak-builder:129186): flatpak-builder-WARNING **: 13:59:35.858: Unknown property sha for type BuilderSourceGit`

```suggestion
    commit: 827b94496f09f0d0bea14e0d2f44bdf0496b1f76
```